pym/bob/cmds/jenkins.py

```diff
@@ -982,7 +983,15 @@ def getJobInternalName(self, step):
         return self.__regexJobName.sub('_', self.getJobDisplayName(step)).lower()
 
 
-def _genJenkinsJobs(step, jobs, nameCalculator, archiveBackend, seenPackages):
+def _genJenkinsJobs(step, jobs, nameCalculator, archiveBackend, seenPackages, allVariantIds = set(),
+                    shortdescription=False):
```

Comment:
No need for default parameters IMHO. This function is internal to the module and all call sites can be adjusted. BTW, the `allVariantIds` default parameter is dangerous: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments